addons/website/static/src/components/configurator/configurator.js

```diff
@@ -200,6 +200,20 @@ class PaletteSelectionScreen extends Component {
         this.logoInputRef.el.click();
     }
 
+    async svgToPng(logo) {
+        return new Promise ((resolved, _) => {
```

Comment:
- There should not be a space after a constructor name
- `resolve` not `resolved`
- You don't need the second argument, just don't name it. And you can get rid of parenthesis and write:

```javascript
return new Promise(resolve => {
```

Reply:
done